parser/prism/Translator.cc

```diff
@@ -1227,7 +1227,7 @@ unique_ptr<parser::Node> Translator::patternTranslate(pm_node_t *node) {
             NodeVec sorbetElements{};
             sorbetElements.reserve(1 + prismMiddleNodes.size() + (prismTrailingSplat != nullptr ? 1 : 0));
 
-            if (prismLeadingSplat != nullptr && PM_NODE_TYPE_P(prismLeadingSplat, PM_SPLAT_NODE)) {
+            if (prismLeadingSplat != nullptr) {
```

Comment:
The `type` field was removed, and the `left` field is now [always a pm_splat_node](https://ruby.github.io/prism/c/structpm__find__pattern__node.html).